entrypoint.sh

```diff
@@ -1,7 +1,21 @@
 #!/bin/bash
-
 set -o errexit
 
+# init /srv if empty
+DIST_FILE=/srv/pmm-distribution
+if [ ! -f $DIST_FILE ]; then
+    echo "File $DIST_FILE doesn't exist. Initizlize /srv..."
+    echo docker > $DIST_FILE
+    mkdir -p /srv/{clickhouse,grafana,logs,postgres,prometheus,nginx,victoriametrics}
+    chown grafana:grafana /srv/grafana
+    chown pmm:pmm /srv/{victoriametrics,prometheus,logs}
+    chown postgres:postgres /srv/postgres
+    echo "Generate self-signed certificates for nginx"
+    bash /var/lib/cloud/scripts/per-boot/generate-ssl-certificate
+    echo "Init Postgres"
+    su postgres -c "/usr/pgsql-11/bin/initdb -D /srv/postgres"
+fi
```

Comment:
would be good to have PMM admin password here as well. As this is an entry point, probably some more standard env name should be chosen, or probably mount point with a pass file ?

https://github.com/percona/percona-helm-charts/pull/132/files#diff-adad060883d938d361b0636b890c5e0a39ff6a9670b6ec9d06d04165293e1019R103